APE1.rst

```diff
@@ -1,11 +1,11 @@
 APE Purpose and Process
 -----------------------
 
-author: Perry Greenfield
+author: Perry Greenfield, Lia Corrales
 
 date-created: 2013 October 11
 
-date-last-revised: 2013 November 8
+date-last-revised: 2020 July 22
 
 date-accepted: 2013 November 8
```

Comment:
Hmm, should there be an extra field called `date-revision-accepted` or something? Seems weird to have `date-accepted` to be 7 years before revision date.

Reply:
That's basically ``date-last-revised`` right?

Reply:
I think `date-last-revised` is OK.

Reply:
I also prefer just leaving it as `date-last-revised`

Reply:
Seems there is consensus here so resolving